ci/scripts/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -eux
+
+cwd=$(pwd)
+
+pushd $cwd/dp-reporter-client
+  make build && mv build/dp-reporter-client $cwd/build
+  cp Dockerfile.concourse $cwd/build
```

Comment:
```suggestion
  make build
```

Reply:
Latest push should have these changes in now.